<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileExtensionsDialog</class>
 <widget class="QDialog" name="FileExtensionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор расширений файлов</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbox_Cpp">
     <property name="title">
      <string>С++</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="ext_h">
        <property name="text">
         <string>.h</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ext_cpp">
        <property name="text">
         <string>.cpp</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Cpp">
        <property name="text">
         <string>Включить</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ext_txt">
     <property name="text">
      <string>.txt</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Свои разрешения:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_CustomExtentions"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_CleanCustomExt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bb_Control">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bb_Control</sender>
   <signal>accepted()</signal>
   <receiver>FileExtensionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_Control</sender>
   <signal>rejected()</signal>
   <receiver>FileExtensionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Cpp</sender>
   <signal>clicked(bool)</signal>
   <receiver>ext_h</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Cpp</sender>
   <signal>clicked(bool)</signal>
   <receiver>ext_cpp</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_CleanCustomExt</sender>
   <signal>clicked()</signal>
   <receiver>le_CustomExtentions</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
